src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from "axios";
+import Gallery from "react-photo-gallery";
+import { shuffleArray } from "utility/Helpers";
+
+const PhotoGallery = () => {
+  const [images, setImages] = React.useState<any[]>([]);
+
+  async function fetchImages() {
+    let response = await axios.get(
+      "https://google-photos-album-demo2.glitch.me/SkVei5N56poqTh8g8"
+    );
+    console.log(response.data);
+    if (response.data && response.data.length > 0) {
+      const shuffledResponseData = shuffleArray(response.data);
```

Comment:
I didn't want our images to look repeated if they're displayed by order of date so I added a utility function to randomly shuffle the elements in any array - in our case we are just using it to shuffle the elements inside the `response.data` array that we are fetching from our shared gallery album on Google Photos: https://photos.google.com/u/2/share/AF1QipMOWdbKrgjqo81b9yCTUika1OE-Me4u9cUTj8aVNenoSgyC2ckLwp2dA3OZUt8vog?key=ZWhsYnlEeDZmNHJRVlBoMG16Wl9JQkJYZXhjRXVn